packages/data-stores/src/site/actions.ts

```diff
@@ -119,7 +125,21 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		return success;
 	}
 
+	function* setSiteTitle( title: string, siteId: number = window._currentSiteId ) {
+		yield wpcomRequest( {
+			path: `/sites/${ encodeURIComponent( siteId ) }/settings`,
+			apiVersion: '1.4',
+			body: {
+				blogname: title,
+			},
+			method: 'POST',
+		} );
```

Comment:
Let's add a comment to mention this can be extracted in the future to a more generic `updateSite` action that will take `siteId` and `updates` object to be sent as `body`. That could be useful if we need to update some other property using `Site` data-store.